app/Http/Controllers/ProjectController.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Client;
+use App\Models\Project;
+use App\Models\User;;
+use Auth;
+use App\Http\Requests\ProjectStoreRequest;
+
+
+class ProjectController extends Controller
+{
+
+    public $users=[];
+    public $allowedUsers=[]; /* has admin or user rule */
+    public $clients=[];
+    public function __construct()
+    {
+        $users=User::with('roles')->orderBy('name')->get();
```

Comment:
@faxunil please format the code according to PSR standards, there should be space for `=` sign, like `$users = [];` and not `$users=[];`, your code is pretty hard to read.